lib/spack/spack/build_systems/cmake.py

```diff
@@ -83,14 +83,21 @@ def build_type(self):
         return 'RelWithDebInfo'
 
     @property
-    def root_cmakelists_dir(self):
-        """Returns the location of the root CMakeLists.txt
+    def cmakelists_directory(self):
+        """The relative path to the directory containing CMakeLists.txt
```

Comment:
It's not obvious to me what this path is relative *to*?  The staging directory or the expanded tarball or ....  Are there a few choice words that can be added to make it clear to those who aren't already in the know?